ui/imports/shared/views/NetworkSelector.qml

```diff
@@ -119,7 +122,7 @@ Item {
                 isBridgeTx: root.isBridgeTx
                 errorType: root.errorType
                 weiToEth: function(wei) {
-                    if(selectedAsset !== undefined)
+                    if(!!selectedAsset && selectedAsset !== undefined)
```

Comment:
shouldn't the first condition be enough?

Reply:
both are needed else I get an undefined error